tests/sync/test_medicaid_eligibility_api.py

```
@@ -0,0 +1,695 @@
+        esrd=False,
+        als=False,
+    )
+    base.update(overrides)
+    return base
+
+
+class TestExpansionAdultFlow(SimpleTestCase):
+    """MAGI adults in expansion states."""
+
+    def test_low_income_expansion_adult_is_eligible_2025(self):
+        eligible_2025, _, _, _, missing = is_eligible(**_answers())
```

Ruff (0.16.1)

[warning] Unpacked variable `missing` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)